.clang-format

```diff
@@ -1,49 +1,49 @@
-﻿---
+---
 AlignAfterOpenBracket: Align
-AlignConsecutiveMacros: 'true'
-AlignConsecutiveAssignments: 'false'
-AlignConsecutiveDeclarations: 'false'
+AlignConsecutiveMacros: true
```

Comment:
According to schema clang format expects booleans/numbers instead of strings: http://clang.llvm.org/docs/ClangFormatStyleOptions.html#configurable-format-style-options